praktikumsplaner-frontend/frontend/src/composables/rules.ts

```diff
@@ -6,7 +6,8 @@ export function useRules() {
 
     function maxLengthRule(length: number, message = "error") {
         return (value: string | null | undefined) =>
-            (value && value.length <= length) || message;
+            ((value || value?.trim() == "") && value.length <= length) ||
```

Comment:
what is this trim comparison used for?

Reply:
This could be done without trim, but it basically also allows blank values. MaxLengthRule should only fail when the length is too long, not too little